### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by polybuildr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # paniLog
 The "who last filled the water bottles?" repository.
 
-#Contribution
-Contribution is simple. First fork the repository. Then come to room E-11, Palash Nivas (Old Boys Hostel) IIIT-H, Gachibowli Hyderabad-500032. You'll find 2 blue coloured bottles. Fill those bottles with RO water (the closest source is South Mess (OBH-Ground Floor Mess)). Keep the water bottles back. If there is a `+` symbol in log file, remove the `+`. Is there isn't add a `+` after the dot which is already there. Commit and send us a pull request. If we find water in the bottles, we will accept the pull request. Simple right. :D
+# Contribution
+
```
Comment (hharchani) [inline README.md:5]: Was this newline really required?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # paniLog
 The "who last filled the water bottles?" repository.
 
-#Contribution
-Contribution is simple. First fork the repository. Then come to room E-11, Palash Nivas (Old Boys Hostel) IIIT-H, Gachibowli Hyderabad-500032. You'll find 2 blue coloured bottles. Fill those bottles with RO water (the closest source is South Mess (OBH-Ground Floor Mess)). Keep the water bottles back. If there is a `+` symbol in log file, remove the `+`. Is there isn't add a `+` after the dot which is already there. Commit and send us a pull request. If we find water in the bottles, we will accept the pull request. Simple right. :D
+# Contribution
```
Comment (hharchani) [inline README.md:4]: Was this space really required?

**Review by hharchani — CHANGES_REQUESTED** [review]
Look good overall. Refer to inline comments.

File: README.md (hunk 2 above)
Comment (polybuildr) [reply README.md:4]: Yes.

File: README.md (hunk 1 above)
Comment (polybuildr) [reply README.md:5]: Yes.

**Review by polybuildr — COMMENTED** [review]
LGTM.

**Review by hharchani — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ParthKolekar) [reply README.md:5]: Yes.

File: README.md (hunk 2 above)
Comment (ParthKolekar) [reply README.md:4]: Yes.

**Comment by ParthKolekar** [comment]
Yes.

**Pull request merged**